kafkabase/kafka_config.go

```diff
@@ -28,6 +28,7 @@ type KafkaConfig struct {
 	KafkaFetchMessageMaxBytes int    `mapstructure:"KAFKA_FETCH_MESSAGE_MAX_BYTES" default:"1048576"`
 
 	KafkaRetryTopicSegmentBytes              int    `mapstructure:"KAFKA_RETRY_TOPIC_SEGMENT_BYTES" default:"104857600"`
+	KafkaRetryTopicRetentionHours            int    `mapstructure:"KAFKA_RETRY_TOPIC_RETENTION_HOURS" default:"168"`
```

Comment:
https://github.com/jitsucom/bulker/blob/c0fb0230b96ce8ee4b559d6f4822f611cae095ac/.docs/server-config.md?plain=1#L203-L207

Reply:
Thank you!

Actually 7 days is too high value for that. When doing retries, retry consumer re-produces messages that are not ready yet. There is no point of having that retention higher than `KafkaTopicRetentionHours`

Reply:
Got it, I used the default value from the docs. What value would you recommend for this? Should we set it to match the `KafkaTopicRetentionHours` value (2 days)?"

Reply:
2 days is ok

Reply:
Done, I've updated the default value to 2 days.